spec_test.go

```diff
@@ -126,7 +126,10 @@ func TestSpecYamlEncoding(t *testing.T) {
 				CIDR:              "10.0.0.0/16",
 				PrivateSubnetCIDR: "10.0.0.0/19",
 				PublicSubnetCIDR:  "10.0.128.0/20",
-
+				RouteTableNames: []string{
+					"cluster_private_0",
+					"cluster_private_1",
```

Comment:
Can those names be just generated in aws-operator? Are they connected to something?

Reply:
We generate these in aws-terraform, I don't want to replicate the name generation